coss/home/models.py

```diff
@@ -20,36 +22,80 @@ class Meta:
 class AboutSnippet(models.Model):
     title = models.CharField(default='', max_length=40)
     header_text = fields.RichTextField(blank=True)
-    body_title = models.CharField(blank=True, default='', max_length=40)
-    body_text = fields.RichTextField(blank=True)
-    info_title = models.CharField(blank=True, default='', max_length=40)
-    info_text = fields.RichTextField(blank=True)
-    bottom_cta_title = models.CharField('Title', blank=True, default='', max_length=40)
-    bottom_cta_text = models.CharField('Text', blank=True, default='', max_length=40)
-    bottom_cta1_text = models.CharField('Text', blank=True, default='', max_length=20)
-    bottom_cta1_link = models.URLField('Link', blank=True, default='')
-    bottom_cta2_text = models.CharField('Text', blank=True, default='', max_length=20)
-    bottom_cta2_link = models.URLField('Link', blank=True, default='')
+    body_title = models.CharField(verbose_name='Title', blank=True, default='',
+                                  max_length=40)
```

Comment:
Let's be consistent, in other places the title holds `100 chars`.

Reply:
The above applies to all the title fields that exist in this PR.

Reply:
All `title` fields have the same limit. All `text` fields are either unlimited or are limited based on design (eg. buttons can only have really short text).